### PR TITLE
Update _last_active_task_time if robot is executing task

### PR DESCRIPTION
**Pull request by cwrx777**
### Fixed bug
Reference: open-rmf/rmf#516

### Fix applied
Set `_last_active_task_time` to `now` if the robot is currently doing task.

### CODE REVIEW

File: rmf_fleet_adapter/src/rmf_fleet_adapter/agv/RobotContext.cpp
Comment (Yadunund) [inline rmf_fleet_adapter/src/rmf_fleet_adapter/agv/RobotContext.cpp:1499]: Nit: Unnecessary newline.
```suggestion
```

File: rmf_fleet_adapter/src/rmf_fleet_adapter/agv/RobotContext.cpp
Comment (Yadunund) [inline rmf_fleet_adapter/src/rmf_fleet_adapter/agv/RobotContext.cpp:1504]: Some style guides may complain if we have an `else if` block without an `else` so better if we can include something like.
```
else
{
  continue;
}

```

**Review by Yadunund — CHANGES_REQUESTED** [review]

**Review by Yadunund — APPROVED** [review]
LGTM from me. Will wait for an approval from @mxgrey before merging.

**Review by mxgrey — APPROVED** [review]
Makes sense, thanks for the fix!

**Pull request merged**
